From 4bc17c1aefa3a604b01edc60ffa71e56fc5c701e Mon Sep 17 00:00:00 2001
From: Yu Zhang <yu.c.zhang@intel.com>
Date: Tue, 27 Jan 2015 15:38:42 +0800
Subject: [PATCH 12/18] resize MAX_NR_IO_RANGES to 512

MAX_NR_IO_RANGES is enlarged to 512. But this value should better be
configurable if we wanna upstream.

Signed-off-by: Yu Zhang <yu.c.zhang@intel.com>
diff --git a/xen/include/asm-x86/hvm/domain.h b/xen/include/asm-x86/hvm/domain.h
index f990ba5..492bc92 100644
--- a/xen/include/asm-x86/hvm/domain.h
+++ b/xen/include/asm-x86/hvm/domain.h
@@ -49,7 +49,7 @@ struct hvm_ioreq_vcpu {
 };
 
 #define NR_IO_RANGE_TYPES (HVMOP_IO_RANGE_PCI + 1)
-#define MAX_NR_IO_RANGES  256
+#define MAX_NR_IO_RANGES  512
 
 struct hvm_ioreq_server {
     struct list_head       list_entry;
